third_party/directory_locations.json

```diff
@@ -167,11 +167,11 @@
     },
     {
         "source": "cookiesjs/cookies.js",
-        "destination": "devsite/source/assets/js/libs/cookies.js"
+        "destination": "devsite/source/assets/js/libs/"
     },
     {
         "source": "cookiesjs/cookies.js",
-        "destination": "devsite/source/_js/libs/cookies.min.js"
+        "destination": "devsite/source/_js/libs/"
```

Comment:
This is going to fail to produce a cookies.min.js - you probably need to copy the file for this to work.